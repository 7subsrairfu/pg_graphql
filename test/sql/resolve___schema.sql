select jsonb_pretty(
    gql.resolve($$
        query IntrospectionQuery {
          __schema {
            queryType {
              name
            }
            mutationType {
              name
            }
            types {
              kind
              name
            }
            directives {
              name
              description
              locations
            }
          }
        }
    $$)
);
