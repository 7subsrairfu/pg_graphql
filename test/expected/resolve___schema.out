select jsonb_pretty(
    gql.dispatch($$
        query IntrospectionQuery {
          __schema {
            queryType {
              name
            }
            mutationType {
              name
            }
            types {
              kind
              name
            }
            directives {
              name
              description
              locations
            }
          }
        }
    $$)
);
                   jsonb_pretty                    
---------------------------------------------------
 {                                                +
     "data": {                                    +
         "__schema": {                            +
             "types": [                           +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "Account"            +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "AccountConnection"  +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "AccountEdge"        +
                 },                               +
                 {                                +
                     "kind": "SCALAR",            +
                     "name": "BigInt"             +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "Blog"               +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "BlogConnection"     +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "BlogEdge"           +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "BlogPost"           +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "BlogPostConnection" +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "BlogPostEdge"       +
                 },                               +
                 {                                +
                     "kind": "ENUM",              +
                     "name": "BlogPostStatus"     +
                 },                               +
                 {                                +
                     "kind": "SCALAR",            +
                     "name": "Boolean"            +
                 },                               +
                 {                                +
                     "kind": "SCALAR",            +
                     "name": "DateTime"           +
                 },                               +
                 {                                +
                     "kind": "SCALAR",            +
                     "name": "Float"              +
                 },                               +
                 {                                +
                     "kind": "SCALAR",            +
                     "name": "ID"                 +
                 },                               +
                 {                                +
                     "kind": "SCALAR",            +
                     "name": "Int"                +
                 },                               +
                 {                                +
                     "kind": "SCALAR",            +
                     "name": "JSON"               +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "Mutation"           +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "PageInfo"           +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "Query"              +
                 },                               +
                 {                                +
                     "kind": "SCALAR",            +
                     "name": "String"             +
                 },                               +
                 {                                +
                     "kind": "SCALAR",            +
                     "name": "UUID"               +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "__Directive"        +
                 },                               +
                 {                                +
                     "kind": "ENUM",              +
                     "name": "__DirectiveLocation"+
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "__EnumValue"        +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "__Field"            +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "__InputValue"       +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "__Schema"           +
                 },                               +
                 {                                +
                     "kind": "OBJECT",            +
                     "name": "__Type"             +
                 },                               +
                 {                                +
                     "kind": "ENUM",              +
                     "name": "__TypeKind"         +
                 }                                +
             ],                                   +
             "queryType": {                       +
                 "name": "Query"                  +
             },                                   +
             "directives": [                      +
             ],                                   +
             "mutationType": {                    +
                 "name": "Mutation"               +
             }                                    +
         }                                        +
     },                                           +
     "errors": [                                  +
     ]                                            +
 }
(1 row)

